.github/no-response.yml

```diff
@@ -0,0 +1,11 @@
+# Configuration for probot-no-response - https://github.com/probot/no-response
+
+# Number of days of inactivity before an Issue is closed for lack of response
+daysUntilClose: 14
```

Comment:
@guolinke 
Sorry, haven't had a chance to read about bots yet. But at present I'm personally trying to give from 25 to 30 days for providing any feedback after the last ping. Maybe we can increase this value for at least 21 days?..

Reply:
^ I agree with this. I think 21 or even 30 days would be better. I think the main goal of the bot should be to automate some of the work we do to clean up things that are months old